gsoc2018-bharat/src/combine.py

```diff
@@ -0,0 +1,82 @@
+import os, sys
```

Comment:
It's not great to use commenting / uncommenting as a way to control the logic of your program, might be good to use other resources

Reply:
No. That does not alter the flow of control. Nor is it associated with the logic. It was just to count the unique resources in the entire corpus. It also includes the resources that do not have a description but are mentioned nonetheless.